File: bot/commands/util/vote.js
exports.run = async (bot) => {
  bot.registerCommand("vote", async (message, args) => {
    await bot.createMessage(message.channel.id, `${bot.emojis.blank} **|** You can vote for Atlantis here:\n${bot.emojis.blank} **|** https://top.gg/bot/688911718788628496/vote`)
  }, {
    cooldown: 3000,
    cooldownMessage: "Whoa there slow down, the cooldown is 3 seconds!"
  })
}

File: src/main/java/dastrangeboi/atlantis/Commands/Vote.java
package dastrangeboi.atlantis.Commands;

import dastrangeboi.atlantis.Bot;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

public class Vote {
    public static void run(GuildMessageReceivedEvent message) {
        message.getChannel().sendMessage("You can vote for Atlantis here:\nhttps://top.gg/bot/688911718788628496/vote").queue();
    }
}
